crates/rattler_package_streaming/src/reqwest/tokio.rs

```diff
@@ -159,6 +159,9 @@ pub async fn extract_conda(
             if (zip_error.contains(DATA_DESCRIPTOR_ERROR_MESSAGE)) =>
         {
             tracing::warn!("Failed to stream decompress conda package from '{}' due to the presence of zip data descriptors. Falling back to non streaming decompression", url);
+            if let Some(reporter) = &reporter {
+                reporter.on_download_complete();
+            }
             let new_reader =
                 get_reader(url.clone(), client, expected_sha256, reporter.clone()).await?;
             crate::tokio::async_read::extract_conda_via_buffering(new_reader, destination).await
```

Comment:
Do you think I should also be reporting the conclusion of this download?

Reply:
If the called function doesnt so this internally then that makes sense yeah!

Reply:
Updated!